### PR TITLE
Remove unnecessary includes in the BPF program

### PR DESCRIPTION
**Pull request by javierhonduco**
to simplify the code. Fixes the build as well.

Signed-off-by: Francisco Javier Honduvilla Coto <javierhonduco@gmail.com>

### CODE REVIEW

**Review by kakkoyun — APPROVED** [review]

**Comment by javierhonduco** [comment]
```
> make[1]: *** 3rdparty/libbpf/src: No such file or directory.  Stop.
```

I guess we need to manually initialise the submodule?

**Comment by kakkoyun** [comment]
> ```
> > make[1]: *** 3rdparty/libbpf/src: No such file or directory.  Stop.
> ```
> 
> I guess we need to manually initialise the submodule?

Yes, one needs to run `git submodule update --init`

File: Makefile [hunk 1]
```diff
@@ -40,7 +40,7 @@ BPF_HEADERS := 3rdparty/include
 BPF_BUNDLE := $(OUT_DIR)/tiny-profiler.bpf.tar.gz
 
 LIBBPF_SRC := 3rdparty/libbpf/src
-LIBBPF_HEADERS := $(OUT_DIR)/libbpf/$(ARCH)/usr/include
+LIBBPF_HEADERS := 3rdparty/libbpf/src
```
Comment (kakkoyun) [inline Makefile]: This was actually correct. No need to change.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -70,6 +70,9 @@ jobs:
           sudo apt-get update -y
           sudo apt-get install -yq libelf-dev zlib1g-dev
 
+      - name: Initialise submodules
+        run: git submodule update --init
```
Comment (kakkoyun) [inline .github/workflows/build.yml]: This is also exists in few lines below

File: Makefile (hunk 1 above)
Comment (javierhonduco) [reply Makefile]: Interesting, it did not compile on my machine without this. Let me try again

**Comment by kakkoyun** [comment]
@javierhonduco I think I have fixed several build issues on the main branch. Just try to rebase.
